provider/terraform.rb

```diff
@@ -293,5 +293,9 @@ def extract_identifiers(url)
     def id_format(object)
       object.id_format || object.self_link_uri
     end
+
+    def conversion?
```

Comment:
It seems strange for a parent class to have a method asserting that it is not an overridden version of itself. Could this be done with class checking or compiler type checking?